src/core/client/admin/routes/Moderate/Queue/Queue.tsx

```diff
@@ -63,8 +60,15 @@ const Queue: FunctionComponent<Props> = ({
   const [conversationModalVisible, setConversationModalVisible] =
     useState(false);
   const [conversationCommentID, setConversationCommentID] = useState("");
+  const [hasModerated, setHasModerated] = useState(false);
   const memoize = useMemoizer();
 
+  useEffect(() => {
```

Comment:
So we can register hotkeys for the first comment without immediately pulling focus

Reply:
Might be useful also to add this comment to the code.

Reply:
✅